capstone/src/main/java/com/google/sps/search/servlets/NameTestDataServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.search.servlets;
+
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.Objects.User;
+import com.google.sps.search.CsvReader;
+
+@WebServlet("/name-test-data")
+public class NameTestDataServlet extends HttpServlet {
+
+  @Override
+  public void init() {
+    CsvReader reader = new CsvReader();
+    List<String> userInfo = reader.parseCorpus("../../data/user_data.csv");
```

Comment:
parseCorpus returns null if there was an issue reading the data, you should make sure that userInfo is not null before continuing